website/content/en/docs/upgrading-sdk-version/v1.0.0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: v1.0.0
+weight: 999000000
+---
+
+## Upgrade your project from version "2" to "3-alpha"
+
+The SDK's default Go plugin no longer supports OLM- or scorecard-related project files nor writes a `plugins` PROJECT field for projects scaffolded previously with `operator-sdk init --project-version=2`, Please migrate to project version "3-alpha" for support of these features by adding the following to your `PROJECT` file:
+```yaml version: "3-alpha" # Updated from "2" projectName: <output of $(basename $(pwd))> layout: go.kubebuilder.io/v2 plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+```
```

Comment:
````suggestion

```yaml
version: "3-alpha" # Updated from "2"
projectName: <output of $(basename $(pwd))>
layout: go.kubebuilder.io/v2 plugins:
  go.sdk.operatorframework.io/v2-alpha: {}
```
````